Network/network-exchange-connectivity-matrix-configuration-guide.md

```diff
@@ -21,7 +21,7 @@ New Jersey, NJ|NJ2|NJ2<br>NJ2X|NY1
 Seattle, WA|SE2|SE2|WA1
 London, UK|LO1|LO1|GB3
 London, UK|LO6|LO6|GB1
-Frankfurt, DE|FR6|FR6|DE1
+Frankfurt, DE|FR6|FR6|DE1|DE3
```

Comment:
this should be DE1<br>DE3 right? you want to list both sides in the same table box. its not rendering properly as you submitted

Reply:
```
DE1<br>DE3
```
sorry